tools/ci_build/build.py

```diff
@@ -894,7 +889,7 @@ def generate_build_tree(
         "-Donnxruntime_USE_VITISAI=" + ("ON" if args.use_vitisai else "OFF"),
         "-Donnxruntime_USE_TENSORRT=" + ("ON" if args.use_tensorrt else "OFF"),
         "-Donnxruntime_SKIP_AND_PERFORM_FILTERED_TENSORRT_TESTS="
-        + ("ON" if args.test_all_timeout == "10800" else "OFF"),
+        + ("ON" if (args.test_all_timeout == "10800" and not args.tensorrt_placeholder_builder) else "OFF"),
```

Comment:
would it make sense to only check for placeholder_builder option and not the timeout value?
(idea is with the placeholder builder, we don't incur any builder overhead?)